ToDoListApp/TaskViewController.swift

```diff
@@ -9,10 +9,16 @@ import UIKit
 
 class TaskViewController: UIViewController {
 
+    @IBOutlet var label: UILabel!
+    
+    var task: String?
+    
     override func viewDidLoad() {
         super.viewDidLoad()
 
         // Do any additional setup after loading the view.
+        label.text = task
```

Comment:
Try all the features of label

Reply:
textAlignment
textColor
text
numberOflines
lineBreakMode
font